packages/snap/src/keyring.test.ts

```diff
@@ -43,27 +55,41 @@ jest.mock('../src/utils/ethers', () => ({
     new ethers.Wallet(privateKey, ethers.provider),
 }));
 
-jest.mock('uuid', () => ({
-  v4: () => mockAccountId,
-}));
+jest.mock('uuid', () => {
+  return {
+    v4: () => {
+      accountCreationCount += 1;
+      return accountCreationCount === 1
```

Comment:
I think we could add a quick comment that explains that we are mocking `uuid.v4` only for the first call, then next calls with yield a true UUID v4 afterward, WDYT?

Reply:
LGTM! [18da79e](https://github.com/MetaMask/snap-account-abstraction-keyring/pull/51/commits/18da79eb601ac4968c31534dadea2c88fed7a251)